mergify_engine/redis_utils.py

```diff
@@ -119,6 +122,9 @@ class RedisLinks:
     cache_max_connections: typing.Optional[int] = None
     stream_max_connections: typing.Optional[int] = None
     queue_max_connections: typing.Optional[int] = None
+    team_permissions_cache_max_connections: typing.Optional[int] = None
```

Comment:
there's no settings exposed in the config

Reply:
I dropped them and used the cache_max_connections